src/matlab/reactormethods.cpp

```diff
@@ -48,6 +48,13 @@ void reactormethods(int nlhs, mxArray* plhs[],
         case 7:
             iok = reactor_setKineticsMgr(i, int(v));
             break;
+        case 8:
+            // This casting of double to bool is recommended by the MathWorks
```

Comment:
It seems very odd to me to cast a `double` to a `bool` since a `double` can appear to be zero when printed (e.g. during debugging) but not actually cast to `false`. Maybe that's just unfamiliarity with the standard operating procedure in C/C++, but I learned to be skeptical of testing doubles for truthiness (and by extension, cast `double`s to `bool`) and I just wanted to document that this was the expected solution.